src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
+    dev = os.path.realpath(dev)
```

Comment:
@dachary 
Does it really matter for this function? I mean, it is a real device, or a link to the device.
Do you have cases where it is a link to a device, but you still would like the answer to this function to be false? Could be, given my limited knowledge of all the variations on linux devices